### PR TITLE
IPTV-708 Implemented broadcast stub and fixed old base tv source tests

### PR DESCRIPTION
**Pull request by rthompson1991**

### CODE REVIEW

File: static/script-tests/tests/devices/broadcastsource/stubbedsourcetest.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * All rights reserved
+ * Please contact us for an alternative licence
+ */
```
Comment (subsidel) [inline static/script-tests/tests/devices/broadcastsource/stubbedsourcetest.js]: Can we swap this block to the smaller version like in: b169e856ce6cc78d7ff4b60cfa5c3608888c5038 ?

File: static/script/devices/broadcastsource/stubbedsource.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * All rights reserved
+ * Please contact us for an alternative licence
+ */
```
Comment (subsidel) [inline static/script/devices/broadcastsource/stubbedsource.js]: Can we swap this block to the smaller version like in: b169e85 ?

Comment (tsadler1988) [inline static/script/devices/broadcastsource/stubbedsource.js]: Good point Joel - I was meant to tell people about this...will spam everyone now.

Comment (subsidel) [inline static/script/devices/broadcastsource/stubbedsource.js]: Thanks :)

**Comment by subsidel** [comment]
Do we need this branch anymore?

**Pull request merged**